otter/integration/lib/autoscale.py

```diff
@@ -49,7 +49,7 @@ def extract_active_ids(group_status):
 
 def create_scaling_group_dict(
     image_ref=None, flavor_ref=None, min_entities=0, name=None,
-    max_entities=25, use_lbs=None
+    max_entities=25, use_lbs=None, server_name=None
```

Comment:
Non-blocking :: Recommendation: change to `server_name_prefix` or some similarly suggestive term.